deal.II-toolchain/packages/petsc.package

```diff
@@ -2,8 +2,11 @@
 ## PETSc                                                                      ##
 ################################################################################
 
-VERSION=3.19.0
-CHECKSUM=3d8820670f0d67aa04ce41b91f8eae62
+VERSION=3.19.3
+CHECKSUM=b493f0c19c067994ce7e9b5f4d13216c
+
+#VERSION=3.19.0
```

Comment:
You can delete this entry. There is no good reason to install 3.19.0 if you have 3.19.3 or .4

Reply:
Got it.